README.md

```diff
@@ -27,10 +27,19 @@ pre-commit install
 ```
 source .venv/bin/activate
 export DEVELOPMENT_MODE=True
+python manage.py migrate
 #python manage.py startapp polls
 DEBUG=True python manage.py runserver
 ```
 
+- Create a `.env` file in base directory and place secret keys related to google auth there.
+- Go to `django admin`:
+    - Go to  `Sites` table and create a new entry with `127.0.0.1:8000` as value for both `Domain Name` and `Display Name`
+    - Go to `Social applications` table and create a new entry with following credentials:
```

Comment:
Can we just use fixtures for this instead of doing it manually?

How does this work on an pre-existing deployment?

Reply:
Yes, I think we can use fixtures